ccpp/data/GFS_typedefs.meta

```diff
@@ -2244,6 +2244,46 @@
   type = real
   kind = kind_phys
   active = (flag_for_stochastic_skeb_option)
+[spp_wts_pbl]
+  standard_name = weights_for_stochastic_spp_pbl_perturbations
+  long_name = weights for stochastic spp pbl perturbations
+  units = none
```

Comment:
For this and other spp_wts_*, units = 1 should replace units = none, which, according to the rules needed by the parser. See the bottom section talking about units for dimensionless variables: https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst

Reply:
Updated.  Thanks!